resources/js/bootstrap.js

```diff
@@ -1,18 +1,3 @@
-window._ = require('lodash');
```

Comment:
Cleaned to only use axios. Bootstrap and other FE dependencies are imported in layout. As a **future** improvement, we can import all dependencies using laravel mix.